app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2>Update Task</h2>
+<%= form_with model: @task do |f| %>
+  <p>Hit update task button when you are done:</p>
+
+  <%= f.label :action %>
+  <%= f.text_field :action %>
```

Comment:
You've got almost exactly the same code here to render a form as you do in `new`. You should dry this up with a view partial.